addons/website/static/src/snippets/s_image_gallery/options.js

```diff
@@ -46,7 +46,8 @@ options.registry.gallery = options.Class.extend({
             }
         });
 
-        if (this.$('> div:first-child > *:not(div)').length) {
+        const $container = this.$('> .container, > .container-fluid, > .o_container_small');
```

Comment:
div first-child not good enough?

Reply:
no because it causes a bug when there is a video on background. In this case the first div is the span for the video frame and not the container

Reply:
Indeed :+1: